test/pagespeed_test.conf.template

```diff
@@ -1058,6 +1059,59 @@ http {
     pagespeed DisableFilters remove_comments,add_instrumentation;
   }
 
+  # Start remote config servers.
+  server {
+    listen @@SECONDARY_PORT@@;
+    listen [::]:@@SECONDARY_PORT@@;
+    server_name remote-config.example.com;
+    pagespeed FileCachePath "@@SECONDARY_CACHE@@";
+    pagespeed RemoteConfigurationUrl "http://localhost:@@PRIMARY_PORT@@/mod_pagespeed_test/remote_config/remote.cfg?PageSpeed=off";
+    #pagespeed RemoteConfigurationUrl "http://raxcity.com/remote.cfg";
+  }
+  server {
+    listen @@SECONDARY_PORT@@;
+    listen [::]:@@SECONDARY_PORT@@;
+    server_name remote-config-partially-invalid.example.com;
+    pagespeed FileCachePath "@@SECONDARY_CACHE@@";
+    pagespeed RemoteConfigurationUrl "http://localhost:@@PRIMARY_PORT@@/mod_pagespeed_test/remote_config/remote.cfg.partially.invalid";
```

Comment:
Or is the exception in `location /mod_pagespeed_test/remote_config/` enough, in which case we can remove it from L1068 above?